### ISSUE TITLE
Search result cropping is missing ...

### ISSUE THREAD

**Issue by akiessling** [tpwd/ke_search#242]
https://github.com/tpwd/ke_search/blob/952340b9306f16330f1809e3bfec8166f2323101/Classes/Lib/Searchresult.php#L315

I'm doing an Upgrade vom T3 11.5 to 12.4 and upgraded from ke_search 4.6.6 to 5.5.0.
My screenshot tests show that the ... from the end of the search results are missing, due to https://github.com/tpwd/ke_search/pull/185/files

Setting the return value to this fixes the output and restores the behavior. 
```
return $cObj->crop($content, ($this->conf['resultChars'] ?? 0) . '|…|1');
```

The resultChars comes from the flexform, thus i'm not sure in which case it would be not set?

**Comment by christianbltr** [comment]
Good catch!

This is fixed now and will be in version 5.5.2.

"resultChars" can be empty in the following cases:

* It's possible to enter an empty value in the FlexForm.
* Plugin can be inserted via TypoScript, thus no FlexForm is available.

**Comment by akiessling** [comment]
Hi @christianbltr 
thanks for taking care!

> It's possible to enter an empty value in the FlexForm.
But then no cropping should appear and the ... should not be shown? -> thus should have been  ok 😁 

> Plugin can be inserted via TypoScript, thus no FlexForm is available.
Oh, you're right. TBH i never used the plugin that way due to the immense amount of config options in the flexform
